vxquery-core/src/main/java/org/apache/vxquery/functions/builtin-functions.xml

```diff
@@ -144,36 +144,44 @@
                  class="org.apache.vxquery.runtime.functions.index.IndexUpdaterEvaluatorFactory"/>
     </function>
 
+    <!-- fn:delete-index($indexFolder  as xs:string?) as  node()* -->
+    <function name="fn:delete-index">
+        <param name="index-folder" type="xs:string?"/>
+        <return type="node()*"/>
```

Comment:
Return boolean value on whether the function was successful. Also change update-index.